### PR TITLE
Hopefully fix 39 tests by not installing reproject

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -92,7 +94,11 @@ commands =
 [testenv:build_docs]
 changedir = docs
 description = Invoke sphinx-build to build the HTML docs
-extras = dev
+# Be verbose about the extras rather than using dev for clarity
+extras =
+  all
+  test
```
Comment (nabobalis) [inline tox.ini:100]: Should we need this to build our docs?

**Review by nabobalis — APPROVED** [review]

**Comment by kakirastern** [comment]
Oh wow, the exact same trick I saw @nabobalis used before

File: tox.ini [hunk 2]
```diff
@@ -76,9 +76,11 @@ deps =
     figure-!devdeps: matplotlib==3.3.2
 
 # The following indicates which extras_require from setup.cfg will be installed
-# dev is special in that it installs everything
+# We use test and all so we don't install docs when we don't need it (as
+# opposed to dev).
 extras =
-    dev
+    all
+    test
```
Comment (nabobalis) [inline tox.ini:83]:
```suggestion
    tests
```

File: tox.ini (hunk 1 above)
Comment (nabobalis) [inline tox.ini:100]:
```suggestion
```

File: tox.ini [hunk 3]
```diff
@@ -92,7 +94,10 @@ commands =
 [testenv:build_docs]
 changedir = docs
 description = Invoke sphinx-build to build the HTML docs
-extras = dev
+# Be verbose about the extras rather than using dev for clarity
+extras =
+  all
+  docs
```
Comment (nabobalis) [inline tox.ini:100]: Let us see if we need test deps to build our docs.

**Comment by kakirastern** [comment]
```
ImportError: Solar system JPL ephemeris calculations require the jplephem package (https://pypi.org/project/jplephem/)
```

**Comment by Cadair** [comment]
Well I would consider that a bug, because it's not actually in our dependency list. @ayshih should I add a skip if no jplephem on those tests?

**Comment by nabobalis** [comment]
> Well I would consider that a bug, because it's not actually in our dependency list. @ayshih should I add a skip if no jplephem on those tests?

Is it too big to just add to our test dep list?

**Comment by ayshih** [comment]
The problem with adding a skip is that I'm not sure what CI build would actually run those tests then...

**Comment by Cadair** [comment]
I guess we could add it as a test dep. I shall do this.

**Comment by kakirastern** [comment]
Someone with access to RTD will need to kick-start the build again for the RTD check to pass.

**Review by kakirastern — APPROVED** [review]

**Pull request merged**